app/components/pet_avatar_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Renders a User's avatar as image or user's initials
+class PetAvatarComponent < ApplicationComponent
+  option :pet, Types::Instance(Pet), optional: true
```

Comment:
Is there a case we would use this Class without passing in a Pet? The benefit of requiring is it will cause an error if missing from the args.